<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterForm</class>
 <widget class="QWidget" name="RegisterForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="FirstName_Label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>First Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="LastName_Label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Last Name</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="FirstName_Text">
   <property name="geometry">
    <rect>
     <x>162</x>
     <y>60</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Username_Label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLabel" name="RegisterHeader_Label">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>20</y>
     <width>59</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Welcome</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="Username_Text">
   <property name="geometry">
    <rect>
     <x>162</x>
     <y>140</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="Password_Text">
   <property name="geometry">
    <rect>
     <x>162</x>
     <y>180</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Password_Label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Save_Button">
   <property name="geometry">
    <rect>
     <x>112</x>
     <y>240</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="LastName_Text">
   <property name="geometry">
    <rect>
     <x>162</x>
     <y>100</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
